Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
Z
A
920/920 [==============================] - 3535s 4s/step - loss: 0.1095 - binary_accuracy: 0.9712 - val_loss: 0.1371 - val_binary_accuracy: 0.9689
Epoch 2/6
A
Z
A
Z
A
A
  1/920 [..............................] - ETA: 52:35 - loss: 0.1467 - binary_accur  2/920 [..............................] - ETA: 35:53 - loss: 0.1960 - binary_accuracy: 0.9573Z
A
Z
A
  3/920 [..............................] - ETA: 49:49 - loss: 0.1926 - binary_accur  4/920 [..............................] - ETA: 50:15 - loss: 0.1799 - binary_accuracy: 0.9570Z
A
Z
A
  5/920 [..............................] - ETA: 50:30 - loss: 0.1673 - binary_accur  6/920 [..............................] - ETA: 50:40 - loss: 0.1696 - binary_accuracy: 0.9574Z
A
  7/920 [..............................] - ETA: 50:46 - loss: 0.1577 - binary_accuracy: 0.9605Z
A
  8/920 [..............................] - ETA: 50:52 - loss: 0.1463 - binary_accuracy: 0.9639Z
A
  9/920 [..............................] - ETA: 50:55 - loss: 0.1395 - binary_accuracy: 0.9659Z
A
 10/920 [..............................] - ETA: 50:55 - loss: 0.1337 - binary_accuracy: 0.9679Z
A
 11/920 [..............................] - ETA: 50:55 - loss: 0.1326 - binary_accuracy: 0.9685Traceback (most recent call last):
  File "run_model_w_generator_cmd.py", line 201, in <module>
    validation_steps = val_generator.steps_per_epoch() )
  File "/home/kiran/anaconda3/envs/my_env/lib/python3.7/site-packages/keras/legacy/interfaces.py", line 91, in wrapper
    return func(*args, **kwargs)
  File "/home/kiran/anaconda3/envs/my_env/lib/python3.7/site-packages/keras/engine/training.py", line 1418, in fit_generator
    initial_epoch=initial_epoch)
  File "/home/kiran/anaconda3/envs/my_env/lib/python3.7/site-packages/keras/engine/training_generator.py", line 181, in fit_generator
    generator_output = next(output_generator)
StopIteration
